easybuild/easyconfigs/d/DFT-D3/DFT-D3-3.1.1-intel-2015a.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'MakeCp'
+
+name = 'DFT-D3'
+version = '3.1.1'
+
+homepage = 'http://www.thch.uni-bonn.de/tc/index.php?section=downloads&subsection=DFT-D3&lang=english'
+description = """DFT-D3 implements a dispersion correction for density functionals, Hartree-Fock and semi-empirical quantum chemical methods."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['http://www.thch.uni-bonn.de/tc/downloads/%(name)s/data']
+sources = ['dftd3.tgz']
```

Comment:
for non-versioned tarballs, please include a checksum too

(or maybe even require to rename the tarball as well)